tests/unit/components/tab/Tab.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import Button from '@mui/material/Button'
+import { render, screen, fireEvent } from '@testing-library/react'
+
+import Tab from '~/components/tab/Tab'
+
+const children = 'Test text'
+const handleTab = vi.fn()
+
+vi.mock('@mui/material/Button', () => {
+  return {
+    __deModule: true,
+    default: vi.fn(() => {
+      return <Button>Looks like it does not work</Button>
+    })
+  }
+})
+
+describe('Tab component test', () => {

```

Comment:
Also, can you add these scenarios:
- [ ] Applies active tab if it is true
- [ ] Does not apply active tab if it is false